pages/_app.js

```diff
@@ -38,7 +39,9 @@ export default function App({ Component, pageProps }) {
         <CSSReset />
         <Navbar />
         <GlobalStyle />
-        <Component {...pageProps} />
+        <Box mx={[2, 0]}>
```

Comment:
Adds a slight x-margin to mobile pages so they're not scrunched directly against the edge.